test/runtests.jl

```diff
@@ -508,4 +508,13 @@ end
         @btime schreier_sims($(gens(G)))
         # GAP: 17 ms vs 50 ms here
     end
-end
\ No newline at end of file
+end
+
+@testset "Iterate over PrmGroup" begin
+	G = PermGroup(6);
+	K = PrmGroup(rand(G,2));
```

Comment:
use `K1 = PrmGroup([perm"(5,6)", perm"(1,2,3,4,5,6)"]) # Symmetric group on 6 symbols` 
and `@test order(K1) == length(uniq_elements) == 720`

add `K2 = PrmGroup([perm"(3,4,5)", perm"(1,2,3,4,5)"]) # Alternating group on 5 symbols`
of order `60`